packages/components/tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { puikTailwindPreset } = require('@prestashopcorp/puik-tailwind-preset');
+module.exports = {
+  presets: [puikTailwindPreset],
+  content: ['./**/src/*.vue'],
+  plugins: [],
+  darkMode: 'class'
+};
```

Comment:
You don't need to create a new tailwind config if you want to compile tailwind css classes you can directly import the config in the postcss config file

Reply:
Done by @mattgoud